CoreWelcomeView/src/au/gov/asd/tac/constellation/views/welcome/plugins/GettingStartedWelcomePlugin.java

```diff
@@ -0,0 +1,125 @@
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.views.welcome.plugins;
+
+import au.gov.asd.tac.constellation.views.welcome.WelcomePageProvider;
+import au.gov.asd.tac.constellation.views.welcome.WelcomeTopComponent;
+import au.gov.asd.tac.constellation.plugins.PluginInfo;
+import java.awt.Desktop;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.logging.Logger;
+import java.util.logging.Level;
+import javafx.geometry.Pos;
+import javafx.scene.control.Button;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
+import javafx.scene.text.Font;
+import javafx.scene.text.Text;
+import org.openide.util.NbBundle;
+import org.openide.util.lookup.ServiceProvider;
+
+/**
+ * The plugin for the Welcome Page that leads to the Getting Started guides and resources
+ *
+ * @author Delphinus8821
+ */
+
+@ServiceProvider(service = WelcomePageProvider.class, position = 5)
+@PluginInfo(tags = {"WELCOME"})
+@NbBundle.Messages("GettingStartedWelcomePlugin=Getting Started Welcome Plugin")
+public class GettingStartedWelcomePlugin extends WelcomePageProvider {
+    
+    public static final String GETTING_STARTED = "resources/welcome_getting_started.png";
+    final ImageView started = new ImageView(new Image(WelcomeTopComponent.class.getResourceAsStream(GETTING_STARTED)));
+    final Button startedBtn = new Button();
+    
+    private static final Logger LOGGER = Logger.getLogger(GettingStartedWelcomePlugin.class.getName());
+        
+    /**
+     * Get a unique reference that is used to identify the plugin 
+     *
+     * @return a unique reference
+     */
+    @Override
+    public String getName() {
+        return "Getting Started Welcome";
+    }
+    
+    /**
+     * This method describes what action should be taken when the 
+     * link is clicked on the Welcome Page
+     *
+     */
+    @Override
+    public void run() {
+        String url = "https://constellation.readthedocs.io/en/latest/";
+
+        if(Desktop.isDesktopSupported()){
+            Desktop desktop = Desktop.getDesktop();
+            try {
+                desktop.browse(new URI(url));
+            } catch (IOException | URISyntaxException e) {
+                LOGGER.log(Level.WARNING, e.getMessage());
+            }
```

Comment:
Following on from my comment https://github.com/constellation-app/constellation/pull/623#pullrequestreview-546504242 I was able to resolve this by running the open link command in a different thread. Actually, we also have a plugin that I'd prefer to use to open links and I replaced the above code with this and it worked.

```java
PluginExecution.withPlugin(CorePluginRegistry.OPEN_IN_BROWSER)
                    .withParameter(OpenInBrowserPlugin.APPLICATION_PARAMETER_ID, "Open " + getName())
                    .withParameter(OpenInBrowserPlugin.URL_PARAMETER_ID, url)
                    .executeLater(null);
```